File: msg.js
var shell = new ActiveXObject("WScript.Shell");

var message = "CUSTOM TEXT"; //MESSAGE
var title = "Hello World";   //TITLE
var time = 0;                //TIME TO DON'T CLOSE SCREEN

var type = 48 + 0; //BUTTON AND ICON

shell.Popup(message, time, title, type); //START ALL COMANDS

File: msg made in c/msg.c
#include <windows.h>

int main() {
    LPCSTR message = "CUSTOM TEXT"; // MESSAGE
    LPCSTR title = "Hello World";   // TITLE
    UINT type = MB_ICONINFORMATION; // BUTTON AND ICON (48 + 0)

    // TIME 0 TO DON'T CLOSE
    MessageBoxA(NULL, message, title, type);

    return 0;
}